charts/lakefs/templates/dev-postgresql.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.useDevPostgres }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-secret
+  labels:
+    app: postgres
+data:
+  POSTGRES_DB: postgres
+  POSTGRES_USER: lakefs
+  POSTGRES_PASSWORD: lakefs
+  MAX_CONNECTIONS: "1000"
```

Comment:
So for lakefs/fluffy we say dev doesn't need a secret and here we do use one.
If we agree that it is just for local dev and it is not a secret - we can drop this one.

Reply:
removed no config map anymore